File: ws-chat-app-client/src/app/models/chatMessageDto.ts
export class ChatMessageDto {
  user: string;
  message: string;
  receiver: string;

  constructor(user: string, message?: string, recipient?: string) {

    this.user = user;

    if (message){
      this.message = message;
    }

    if(recipient){
      this.receiver = recipient;
    }

  }
}

File: src/main/java/com/my/wschatapp/dto/ChatMessageDto.java
package com.my.wschatapp.dto;

public class ChatMessageDto {

    private String user;

    private String message;

    private String receiver;

    public ChatMessageDto(String user, String message, String receiver) {
        this.user = user;
        this.message = message;
        this.receiver = receiver;
    }

    public String getReceiver() {
        return receiver;
    }

    public void setReceiver(String receiver) {
        this.receiver = receiver;
    }

    public String getUser() {
        return user;
    }

    public void setUser(String user) {
        this.user = user;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    @Override
    public String toString() {
        return "ChatMessageDto{" +
                "user='" + user + '\'' +
                ", message='" + message + '\'' +
                ", receiver='" + receiver + '\'' +
                '}';
    }
}
